apps/services/auth/ids-api/project.json

```diff
@@ -126,6 +126,40 @@
         "parallel": false,
         "cwd": "libs/auth-api-lib"
       }
+    },
+    "ids-get-secrets": {
+      "executor": "nx:run-commands",
+      "options": {
+        "commands": [
```

Comment:
I find this to greedy, as it will completely overwrite the dev´s current `.env.secret`.

Reply:
You're not wrong. Fixed.